### ISSUE TITLE
Global colors lost when I update general settings in wordpress

### ISSUE THREAD

**Issue by jayalove** [elementor/elementor#12562]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

All global colors are lost when I go into Wordpress General Settings. I went into settings - general and changed the name and tagline of the site... then all elementor theme global color settings were lost. I also did it with all plugins deactivated besides elementor and e pro... and changed theme to 2017. still same problem

simple... go to settings, general (general wordpress settings), change something in the general settings and click save.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by cbsa100** [comment]
same here!
waiting for a fix because my site is preety much dead now...

**Comment by arielk** [comment]
Thanks for your report!

Merge to https://github.com/elementor/elementor/issues/12538

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.9

Feel free to update

Thanks!
